schedule/yast/ssh-x@s390x_zvm.yaml

```diff
@@ -0,0 +1,20 @@
+name:           ssh-X
```

Comment:
let's create a folder for those splits so we have: `/yast/ssh-x/ssh-x@s390x_zvm.yaml`

Reply:
OK